packages/ui/__stories__/CheckableSelectField.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react'
```

Comment:
nit: Probably we should wrap long options, wdut?
![image](https://user-images.githubusercontent.com/13898415/127626012-13df0169-ede0-466e-b464-3ad3752c0b32.png)